addon/index.js

```diff
@@ -66,18 +67,32 @@ function updatePrefs() {
   const envName = simplePrefs.prefs.SERVER_ENVIRONMENT;
   const env = (envName in SERVER_ENVIRONMENTS) ?
     SERVER_ENVIRONMENTS[envName] : SERVER_ENVIRONMENTS.production;
+  // List of URLs which can view the installed flag.
+  const whitelistURLs = simplePrefs.prefs.WHITELIST_URLS;
```

Comment:
Oh also, this pref won't get picked up when it changes, because this `updatePrefs()` function [only gets called when the SERVER_ENVIRONMENT pref changes](https://github.com/ckprice/testpilot/blob/issue-463-content-detection/addon/index.js#L106). (Well, and also on browser startup) So, might as well put this `WHITELIST_URLS` list into `SERVER_ENVIRONMENTS`.